src/less/variables.less

```diff
@@ -76,27 +76,33 @@
 // -------------------------
 // grayscale and brand colors used throughout the site.
 
-@primary-brand-green:         #00853f;
-@bright-green:                #8cc63f;
-@body-bg:                     #f2f2f2;
-@primary-body-bg:             #fff;
+@primary-brand-green:         #3CAC30;
+@bright-green:                #3CAC30;
+@body-bg:                     #fff; // @layout6
+@primary-body-bg:             #fff; 
```

Comment:
@mattnickles the old palette had more colors than we have so some of their variables had to be parsed to the same color, thats why body-bg and primary-body-bg have the same value (because Cyclops 2 will be a re-build from scratch we are saving the effort of pointing all the items to the same variables on this version). I think if I update the value to f8f8f8 now it will have a big impact as many items that needed to be fff are pointing to body-bg now.